.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      # Keep the experimental modules up-to-date
+      - dependency-name: "github.com/grafana/xk6-*"
+        dependency-type: "all"
+    target-branch: "master"
+    commit-message:
+      prefix: "Bump "
+      include: "scope"
+    assignees:
+      - "octocat"
```

Comment:
Is it required to pass something? I mean I wondering if we should use an `octocat` handle here

Reply:
No, it's not required. My intention was to highlight these PRs as automatically created, but I'm OK with removing this option.

Reply:
> My intention was to highlight these PRs as automatically created, but I'm OK with removing this option.

Probably, my comment shows that the intention could be easily misinterpreted or at least not clear :sweat_smile: 

If I'm correct, here is an example of the PR created by Dependabot: https://github.com/grafana/jslib.k6.io/pull/104. The first comment already shows that this PR is automatically created, so your concerns are already covered.